src/components/pages/Admin/components/RegistrationRequests/CompanyDetailOverlay/index.tsx

```diff
@@ -346,34 +366,53 @@ const CompanyDetailOverlay = ({
         ) : (
           <DialogContent
             sx={{
-              padding: '0 120px',
-              marginBottom: 5,
+              padding: '30px 110px 56px 110px',
+              width: '100%',
+              marginBottom: activeTab ? 5 : 0,
             }}
           >
             <TabPanel value={activeTab} index={0}>
-              <Box sx={{ width: '100%', margin: '50px 0' }}>
-                <KeyValueView
-                  cols={2.3}
-                  title={t(
+              <Box sx={{ width: '100%' }} className={'company-details'}>
+                <Typography className="title" variant="h4">
+                  {t(
                     'content.admin.registration-requests.overlay.companydatatitle'
                   )}
-                  items={companyData}
-                />
+                </Typography>
+                {companyData?.map((detail) => (
+                  <div className="detail" key={detail.key}>
+                    <Typography variant="body2" className="detail-name">
+                      {detail.key}
+                    </Typography>
+                    <div></div>
+                    <Typography variant="label2" className="detail-text">
+                      {detail.value}
+                    </Typography>
+                  </div>
+                ))}
               </Box>
               <Box ref={modalElement} sx={{ width: '100%' }}>
-                <KeyValueView
-                  cols={2.3}
-                  title={t('content.admin.registration-requests.overlay.docs')}
-                  items={documentData}
-                />
-                <KeyValueView
-                  cols={2.3}
-                  title={t('content.admin.registration-requests.overlay.roles')}
-                  items={companyRoleData}
-                />
+                <Typography className="title" variant="h4">
+                  {t('content.admin.registration-requests.overlay.docs')}
+                </Typography>
+
+                <div className="document-container">
+                  {documentData?.map((detail) => {
+                    return <div key={detail.key}>{detail.value}</div>
```

Comment:
We don't need the curly brackets and return if the arrow function returns a value. Instead of 
```diff
- {documentData?.map((detail) => {
-   return <div key={detail.key}>{detail.value}</div>
- })}
```
write
```diff
+ {documentData?.map((detail) =>
+   <div key={detail.key}>{detail.value}</div>
+ )}
```
(maybe prettier will add another pair of round brackets - but thats okay)

Reply:
Changes done